spv/sync.go

```diff
@@ -545,8 +634,8 @@ func (s *Syncer) receiveGetData(ctx context.Context) error {
 			if len(txHashes) != 0 {
 				var missing []*wire.InvVect
 				var err error
-				foundTxs, missing, err = s.wallet.GetTransactionsByHashes(ctx, txHashes)
-				if err != nil && !errors.Is(err, errors.NotExist) {
+				foundTxs, missing, err = s.getTransactionsByHashes(ctx, txHashes)
+				if err != nil && !errors.Is(errors.NotExist, err) {
```

Comment:
`errors.Is` expects the first parameter to be the error being checked and the second parameter to be the target error.
```suggestion
				if err != nil && !errors.Is(err, errors.NotExist) {
```